src/alert/alert.scss

```diff
@@ -16,53 +17,64 @@
     color: t(iui-text-color);
     background-color: t(iui-color-background-1);
   }
-
-  &-icon {
-    @include iui-icons-default;
-    flex-shrink: 0;
-    margin-left: $iui-m;
-  }
-
-  &-message {
-    margin: $iui-baseline $iui-m;
-  }
-
-  &-link {
-    @include iui-anchor-underline('on-initial');
-    border-radius: $iui-border-radius;
-    cursor: pointer;
-    margin-left: $iui-s;
-    user-select: none;
-    white-space: nowrap;
-  }
-
-  > .iui-button {
-    margin-left: auto;
-    margin-right: $iui-s;
+  @each $status in positive, negative, warning, informational {
+    $statusColor: if($status == informational, 'primary', $status);
+    &.iui-#{$status} {
+      @include themed {
+        background: linear-gradient(
+          rgba(t(iui-color-foreground-#{$statusColor}-rgb), t(iui-opacity-6)),
+          rgba(t(iui-color-foreground-#{$statusColor}-rgb), t(iui-opacity-6))
+        ),
+          linear-gradient(t(iui-color-background-1), t(iui-color-background-1));
+        border-color: t(iui-color-foreground-#{$statusColor});
+      }
+    }
   }
 
   &.iui-sticky {
     border-radius: 0;
-    position: absolute;
+    position: fixed;
```

Comment:
hm... Wondering.. This will always make it fixed relatively to the viewport.
When we say `sticky`, I'd expect possibility to stick it relative to certain element.

Reply:
@FlyersPh9 does our design system limit sticky alert to be only in page? or it can be in modal or just any container?

Reply:
The only thing I can find that might be relevant is this 2 year old mockup that shows an alert fixed to the top of the list in the right pane.  If the user were to scroll, the alert would not scroll with it and would stay in place.
![Screen Shot 2022-05-03 at 4 21 19 PM](https://user-images.githubusercontent.com/849817/166559482-e9bb2ce7-b7e2-4ce5-93f0-300dfaf45ef2.png)

Reply:
developer.bentley.com has this alert:
![image](https://user-images.githubusercontent.com/83585998/166688909-eb47ecf7-1b4a-4fb9-971d-916c904fe108.png)

Reply:
I've set `position: sticky` so it can stick relative to any container. The visual test has been updated to reflect this